presto-product-tests/src/main/java/com/facebook/presto/tests/querystats/QueryStatsClientModuleProvider.java

```diff
@@ -66,8 +69,17 @@ QueryStatsClient getQueryStatsClient(
                     HttpClientConfig httpClientConfig = new HttpClientConfig();
                     httpClientConfig.setKeyStorePath(configuration.getString("databases.presto.https_keystore_path").orElse(null));
                     httpClientConfig.setKeyStorePassword(configuration.getString("databases.presto.https_keystore_password").orElse(null));
+
+                    httpClientConfig.setAuthenticationEnabled(configuration.getBoolean("databases.presto.cli_kerberos_authentication").orElse(false));
+                    httpClientConfig.setKerberosPrincipal(configuration.getString("databases.presto.cli_kerberos_principal").orElse(null));
```

Comment:
We do get a error from the code ( I think from JettyHttpClient). I definitely saw it in my testing